business_rules/utils.py

```diff
@@ -126,17 +153,12 @@ def validate_conditions(input_conditions, rule_schema):
                     for k, v in input_conditions.iteritems():
```

Comment:
I see that `k` is a throw away variable here. Is it not best practise (like I have seen in Platform) to use a `_` to explicitly show this?